Classes/Enemy.js

```diff
@@ -32,7 +32,7 @@ module.exports = class Enemy extends Combatant {
 	 * @param {object} actionsInput
 	 * @param {string} actionsInput.name
 	 * @param {"Fire" | "Water" | "Wind" | "Earth" | "Untyped" | "@{adventure}" | "@{adventureOpposite}"} actionsInput.element
-	 * @param {boolean} actionsInput.isPriority
+	 * @param {integer} actionsInput.priority
```

Comment:
```suggestion
	 * @param {number} actionsInput.priority
```
Although we could label the type `integer` probably better to use the actual JS type of `number`.